Sources/Orbit/Components/Text.swift

```diff
@@ -106,15 +106,7 @@ public struct Text: View {
     }
 
     var attributedTextScaledSize: CGFloat {
-        size.value * sizeCategory.ratio * attributedTextScaledSizeAdjustment
-    }
-
-    var attributedTextScaledSizeAdjustment: CGFloat {
-        switch sizeCategory.ratio {
-            case ..<1:      return 1.07
-            case 1:         return 1
-            default:        return 0.94
-        }
+        size.value * sizeCategory.ratio
```

Comment:
It would be tricky, as it might not match sizing of the custom font which system does automatically (such initializer for system font is available for iOS14+ only). Sizes of native swiftui text and UIKit textview need to align perfectly. Even 0.3 difference in font size would be recognizeable.